### PR TITLE
Ordeal Pets

### PR DESCRIPTION
**Pull request by InsightfulParasite**
## Why It's Good For The Game
I thought it would be fun to have pets that clerks can invest in and the facility can actually feel obligated to protect.
The robots gibbing into oil is more of a non essential immersion thing. Also makes the ground less messy.
## Changelog
:cl:
add: Pet_whistle, makes tamed animals like a freshly fed morsel and chuckles follow you.
add: Imperfect Morsel, feed them and they grow!
add: Disarmed Doubt, accepts user input.
add: Small Chuckles, dont let them near consoles.
/:cl:

### CODE REVIEW

File: ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm [hunk 1]
```diff
@@ -0,0 +1,188 @@
+//Unique Abilities
+	//Improved summon backup. Sorta works through walls.
+/mob/living/simple_animal/hostile/ordeal/summon_backup(distance, exact_faction_match)
```
Comment (dsmith328) [inline ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm:3]: This does not require "distance" if it's not using it in the first place.

Comment (dsmith328) [reply ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm:3]: nor does this even use "extact_faction_match"

Comment (InsightfulParasite) [reply ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm]: This was mostly a weird test thing. Ill remove it.

File: code/modules/mob/living/simple_animal/hostile/ordeal/green.dm [hunk 2]
```diff
@@ -60,6 +60,12 @@
 				H.apply_damage(20, WHITE_DAMAGE, null, H.run_armor_check(null, WHITE_DAMAGE), spread_damage = TRUE)
 			finishing = FALSE
 
+/mob/living/simple_animal/hostile/ordeal/green_bot/spawn_gibs()
+	new /obj/effect/decal/cleanable/oil(drop_location(), src)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/hostile/ordeal/green.dm:64]: This should be a gib spawner.

**Comment by EgorDinamit** [comment]
You accidentally removed 3 abnormalities from the code.

File: ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+	friendly_verb_simple = "smack"
+	speak_emote = list("chuckles", "giggles", "laughs")
+	speak_chance = 5
+	butcher_results = list(/obj/item/food/meat/steak/meatproduct = 1)
+	guaranteed_butcher_results = list(/obj/item/food/meat/steak/meatproduct = 1)
+	stop_automated_movement_when_pulled = 1
+	search_objects = 1
+
+/mob/living/simple_animal/hostile/ordeal/crimson_clown/smallchuckles/Life()
+	. = ..()
```
Comment (EgorDinamit) [inline ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm:162]: This is pointless to have here, since it will run *all* the parent proc stuff.

**Review by dsmith328 — COMMENTED** [review]
I want this tested, seems good

**Review by dsmith328 — APPROVED** [review]

**Pull request merged**